src/test/pybind/test_rbd.py

```diff
@@ -473,6 +473,29 @@ def test_copy3(self):
         self._test_copy(features, self.image.stat()['order'],
                         self.image.stripe_unit(), self.image.stripe_count())
 
+    def test_deep_copy(self):
+        global ioctx
+        global features
+        self.image.write(b'a' * 256, 0)
+        self.image.create_snap('snap1')
+        self.image.write(b'b' * 256, 0)
+        dst_name = get_temp_image_name()
+        dst_features = features | RBD_FEATURE_STRIPINGV2
```

Comment:
Nit: ```features``` might be ```None```

Reply:
Actually, it can't due to `features = int(features) if features is not None else 61` in `setup_module()`. (does it mean we don't test python for old format?)

Anyway, I have removed this line and just pass `features` in the updated version (it looks like enough for testing API).